docs/admin/ha-master-gce.md

```diff
@@ -0,0 +1,159 @@
+---
+assignees:
+- jszczepkowski
+
+---
+
+* TOC
+{:toc}
+
+## Introduction
+
+In kubernetes version 1.5, we added alpha support for replication of kubernetes masters in kube-up/down scripts for GCE.
+This document describes how to use kube-up/down scripts to manage highly available (HA) masters and how HA masters are implemented for GCE case.
+
+## Running HA cluster on GCE
+
+### Starting HA-compatible cluster
```

Comment:
Avoid nesting headings directly beneath one another; that's usually indicative of structural problems.

In this case, "Running HA Cluster on GCE" is uncessary and doesn't add anything. Move the subsequent headers one level up.